### PR TITLE
Features/matsa changing data formai in settings

### PR DESCRIPTION
**Pull request by MarsLviv**
changing data format for files extentions  in QSettings from QString to QStringList by Roman Matsa

### CODE REVIEW

File: src/settings/storeconf.cpp [hunk 1]
```diff
@@ -13,8 +15,8 @@ StoreConf::StoreConf(QWidget * parent)
     : QWidget (parent)
 {
     conFile = getPathToConFile();
-    if (!conFile.size()) // keep all fields with default values, create conf.json with these values
```
Comment (vsakharevych) [inline src/settings/storeconf.cpp:16]: To my opinion it is not user-friendly approach to prompt path to config file when app is launched.
I would suggest to provide user with an opportunity to set settings during run time

File: src/settings/storeconf.h [hunk 2]
```diff
@@ -22,8 +22,9 @@ class StoreConf : public QWidget
     QString applicationName         = "Pair Storm";
     QString analizerStyle           = "white";
     QString analizerFontSize        = "12";
-    QString analizerFontName        = "Times New Roman";
-    QString cppExtentions           = "c;cpp;h;hpp";
+    QString analizerFontName        = "Courier";
+    QString cppExtentions           = "c;cpp;h;hpp;json;txt";
+    QStringList cppExtentionsList {"c", "cpp", "h", "hpp", "json", "txt"};
```
Comment (vsakharevych) [inline src/settings/storeconf.h:39]: I would suggest to add method getExtensions to StoreConf class in order to simplify interface

Comment (vsakharevych) [inline src/settings/storeconf.h:39]: Please provide file extensions in format ".c" ".cpp" etc

**Review by vsakharevych — APPROVED** [review]
Please consider my comments

File: src/settings/storeconf.h (hunk 2 above)
Comment (MarsLviv) [reply src/settings/storeconf.h:39]: Done

File: src/settings/storeconf.cpp (hunk 1 above)
Comment (MarsLviv) [reply src/settings/storeconf.cpp:16]: removed this option

File: src/settings/storeconf.h (hunk 2 above)
Comment (MarsLviv) [reply src/settings/storeconf.h:39]: The purpose of this class is to read data from configuration file
 and save it in persistent platform-independent application settings.
 Reach out these data are possible from any point in application using statements:
QSettings settings;
int i = settings.value("fieldName").toString();
where <fieldName> same as in configuration file.

**Review by DANIV18 — APPROVED** [review]

**Pull request merged**